app/assets/javascripts/browsered/web-payments/helpers.js

```diff
@@ -86,7 +86,8 @@ const getGooglePaymentsConfiguration = () => {
     type: 'CARD',
     parameters: {
       allowedAuthMethods: allowedCardAuthMethods,
-      allowedCardNetworks: allowedCardNetworks
+      allowedCardNetworks: allowedCardNetworks,
+      assuranceDetailsRequired: true
```

Comment:
Are you sure this is right? I thought this was a response property, not a request property.

Reply:
Yep if we set assuranceDetailsRequired to true in the request then we get assuranceDetailsRequired in the response. See [here](https://developers.google.com/pay/api/android/reference/request-objects#CardParameters). It's unclear whether this is explicitly required from both the email and the documentation for SCA.

Reply:
I suppose as long as we don’t blow up when we receive it in the response (which I assume you’ve checked) then it’s good to request it.